src/main/kotlin/no/nav/familie/ba/mottak/task/VurderLivshendelseTask.kt

```diff
@@ -331,13 +319,28 @@ class VurderLivshendelseTask(
         oppgaveClient.oppdaterOppgaveBeskrivelse(oppgave, beskrivelse)
     }
 
-    private fun tilBehandlingstema(restUtvidetBehandling: RestUtvidetBehandling?): String {
+    private fun hentUtvidetBehandling(fagsakId: Long): RestUtvidetBehandling? {
+        val minimalRestFagsak = sakClient.hentMinimalRestFagsak(fagsakId)
+        secureLog.info("Hentet minimal rest fagsak: $minimalRestFagsak")
+        val restMinimalBehandling = minimalRestFagsak.behandlinger.firstOrNull { it.aktiv }
+
+        if (restMinimalBehandling == null) {
+            error("Fagsak ${fagsakId} mangler aktiv behandling. Får ikke opprettet VurderLivshendelseOppgave")
+        }
+
+        val restFagsak = sakClient.hentRestFagsak(fagsakId)
```

Comment:
Jeg tenker at du har alt inne i denne. Først må du plukke ut sisteIverksetteBehandling. Sjekk hentSisteBehandlingSomErIverksatt i Behandlingutils i basak. Og så må du sjekke om underkategori er UTVIDET

Reply:
Nja, det blir kanskje ikke helt riktig heller. Siste behandling trenger ikke å være utvidet for at det finnes utvidet på fagsaken :) Og det kan også være at siste behandling er tagget som utvidet. Hva er målet her? Å opprette oppgave om det finnes utvidet andeler, eller om det finnes løpende utvidet andeler? :)

Reply:
@henninghaakonsen Det blir kanskje riktig nok? Løpende andeler er nok best, men da må man nok lage tjenesten i  ba-sak først? Målet er å redusere antall oppgaver man lager på Sivilstand gift. Som den er nå så lages den for alle sakstyper, men er kun relevant hvis søpende sak er utvidet

Reply:
Ja, stememr. Sivilstand er visstnok kun aktuelt dersom personen mottar utvidet barnetrygd

Reply:
mm, skjønner. Vil kanskje være godt nok ja! 👍